Compute/tests/System/V1/SmokeTest.php

```diff
@@ -102,6 +106,115 @@ public function testInsertInstance(): void
         $this->assertEquals(self::$machineType, $instance->getMachineType());
     }
 
+    /**
+     * @depends testInsertInstance
+     */
+    public function testUpdateDescInstanceToEmpty()
+    {
+        $instance = self::$instancesClient->get(
+            self::$name,
+            self::$projectId,
+            self::ZONE
+        );
+        $this->assertEquals("test", $instance->getDescription());
+        $this->assertEquals("0", $instance->getScheduling()->getMinNodeCpus());
+        $instance->setDescription("");
+        $operation = self::$instancesClient->update(self::$name, $instance, self::$projectId, self::ZONE);
+        self::$zoneOperationsClient->wait($operation->getName(), self::$projectId, self::ZONE);
+        $instance = self::$instancesClient->get(
+            self::$name,
+            self::$projectId,
+            self::ZONE
+        );
+        $this->assertEquals("", $instance->getDescription());
+        $this->assertEquals("0", $instance->getScheduling()->getMinNodeCpus());
+    }
+
+    /**
+     * @depends testInsertInstance
+     */
+    public function testUpdateDescInstanceNonAscii()
+    {
+        $instance = self::$instancesClient->get(
+            self::$name,
+            self::$projectId,
+            self::ZONE
+        );
+        $instance->setDescription("тест");
+        $operation = self::$instancesClient->update(self::$name, $instance, self::$projectId, self::ZONE);
+        self::$zoneOperationsClient->wait($operation->getName(), self::$projectId, self::ZONE);
+        $instance = self::$instancesClient->get(
+            self::$name,
+            self::$projectId,
+            self::ZONE
+        );
+        $this->assertEquals("тест", $instance->getDescription());
+    }
+
+    /**
+     * @depends testInsertInstance
+     */
+    public function testInstanceGroupManagers()
+        // We test here: 1)set body field to zero
+        //               2)set query param to zero
+    {
+        $this->markTestSkipped('b/189586033 query params set 0 is ignored');
+        $globalOpClient = new GlobalOperationsClient();
+        $templateClient = new InstanceTemplatesClient();
+        $managersClient = new InstanceGroupManagersClient();
+        $templateName = "gapicphp" . strval(rand(100000, 999999));
+        $managerName = "gapicphp" . strval(rand(100000, 999999));
+        $instance = self::$instancesClient->get(
+            self::$name,
+            self::$projectId,
+            self::ZONE
+        );
+        $templateResource = new InstanceTemplate([
+            'name' => $templateName,
+            'source_instance' => $instance->getSelfLink()
+        ]);
+
+        try{
+            $op = $templateClient->insert($templateResource, self::$projectId);
+            $globalOpClient->wait($op->getName(), self::$projectId);
+            $managerResource = new InstanceGroupManager([
+                'base_instance_name' => "gapicphp",
```

Comment:
nit: use double quotes only when necessary:
```suggestion
                'base_instance_name' => 'gapicphp',
```